src/test/java/org/opensearch/security/support/StreamableRegistryTest.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.security.support;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.net.InetSocketAddress;
+
+public class StreamableRegistryTest {
+
+    StreamableRegistry streamableRegistry = StreamableRegistry.getInstance();
+
+    @Test
+    public void testStreamableTypeIDs() {
+        Assert.assertEquals(1, streamableRegistry.getStreamableID(InetSocketAddress.class));
```

Comment:
Can we add a negative case as well?

Reply:
This test is present to support the dynamically assigned typeIDs for streamables. We're only adding assertions for the streamables we expect to be registered with the respective type codes.

Reply:
Agreed on the positive case, the comment is more about negative test case to ensure the (error) behavior is same as expected when non streamable class is used accidentally?

Reply:
Ack

Reply:
Added.